recommender/recommender.py

```diff
@@ -968,6 +970,23 @@ def studio_view(self, _context=None):  # pylint: disable=unused-argument
         frag.initialize_js('RecommenderXBlock')
         return frag
 
+    def add_xml_to_node(self, node):
+        """
+        Serialize the XBlock to XML for exporting.
+        """
+        node.tag = 'recommender'
+
+        node.set('intro_enabled', str(self.intro_enabled))
+        node.set('DISABLE_DEV_UX', str(self.client_side_settings['DISABLE_DEV_UX']))
+        node.set('ENTRIES_PER_PAGE', str(self.client_side_settings['ENTRIES_PER_PAGE']))
+        node.set('PAGE_SPAN', str(self.client_side_settings['PAGE_SPAN']))
```

Comment:
On a different note, should the client side settings be saved in the XML? If they're set by the client, maybe they shouldn't be included in the export. I'm open to discussion on this -- you know how these attributes are used better than I do.